test/packaging/ansible/roles/package-install/tasks/package-Darwin.yaml

```diff
@@ -13,23 +9,15 @@
 - name: cleanup | Remove files
   command: rm -rf /usr/local/Cellar/newrelic-infra-agent
 
-- name: cleanup | Install agent to fix permissions
```

Comment:
Is this not needed anymore?

Reply:
I run it w/o it and then upgraded the agent and didn't notice any issue in the process, so I decided to remove it for now.